src/editor/editor.js

```diff
@@ -104,6 +104,19 @@ export default class Editor {
 		 */
 		this.t = this.locale.t;
 
+		/**
+		 * Indicates editor initialization status.
+		 * The following statuses are available:
```

Comment:
Still bad. MIssing blank lines before and after this paragraph.

Reply:
BTW, states aren't available. You don't choose one. The editor may be in one of the following states.

Reply:
Actually – "may" is not completely correct to :D "The editor is..."

Reply:
That part comes from CKE4 docs :D